pkg/build/api/types.go

```diff
@@ -55,6 +55,9 @@ type BuildParameters struct {
 
 	// Output describes the Docker image the Strategy should produce.
 	Output BuildOutput `json:"output,omitempty"`
+
+	// Compute resource requirements to execute the build
+	Resources kapi.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
> Both have the exact same meaning.  No limits, no requests.

both will be enforced that way, but one is a hanging chad with guessed intent and which should elicit a warning "hey it looks like you tried to define a quota but never finished, you might want to take a look at that"